spec.html

```diff
@@ -13484,15 +13484,11 @@ <h1>Static Semantics: Contains</h1>
         </emu-alg>
         <emu-grammar>OptionalChain : `?.` IdentifierName</emu-grammar>
         <emu-alg>
-          1. If _symbol_ is a |ReservedWord|, return *false*.
```

Comment:
Gah, thanks. I [noticed this was going to happen](https://github.com/tc39/ecma262/pull/1646#discussion_r345984898) as part of the optional chaining PR and then completely forgot about it when #1519 landed.